_posts/platform/app/2000-01-01-environment.md

```diff
@@ -32,6 +32,12 @@ You should avoid writing any credentials in the files managed by Git.
 
 The following example is to get the value of the `PORT` variable.
 
+#### Shell
+
+```shell
+$PORT
+```
+
```

Comment:
Is it really needed? We don't really have a customer developing a web application in Bash :D 


```suggestion
```